packages/server/database/migrations/20220121600507-addMoreRetroTemplates.ts

```diff
@@ -0,0 +1,667 @@
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Thanksgiving Retrospective 🥧', true),
+    question: 'Left on the Table 🐌',
+    description: 'What do we wish we’d left behind, or should leave behind from now on?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Thanksgiving Retrospective 🥧', true),
+    question: 'Gratitude 🍰',
+    description: 'What are you thankful for?',
+    sortOrder: 3
+  },
+  {
+    templateId: nameToId('Diwali Retrospective 🪔', true),
+    question: 'Diyas',
+    description: 'What do we need to guide us forward or help us find our way?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Diwali Retrospective 🪔', true),
+    question: 'Rangoli',
+    description: 'What practices or processes are bringing positivity?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Diwali Retrospective 🪔', true),
+    question: 'Lakshmi',
+    description:
+      'What do we need to tidy or clean up? What processes or backlog items need purifying?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Diwali Retrospective 🪔', true),
+    question: 'Ravana',
+    description: 'What demons, projects, or problems are cursing us?',
+    sortOrder: 3
+  },
+  {
+    templateId: nameToId('Halloween Retrospective 🎃', true),
+    question: 'Ghosts 👻',
+    description: 'Boo! What projects or issues caught us by surprise?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Halloween Retrospective 🎃', true),
+    question: 'Zombies 🧟',
+    description: 'What feels slow? What are we dragging our feet on?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Halloween Retrospective 🎃', true),
+    question: 'Brains 🧠',
+    description: 'What did we learn? What do we need to learn?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Halloween Retrospective 🎃', true),
+    question: 'Candy 🍬',
+    description: 'Who deserves candy? What went well? Who do you want to give kudos to?',
+    sortOrder: 3
+  },
+  {
+    templateId: nameToId('Questions, Comments, Concerns❓', true),
+    question: 'Questions ❓',
+    promptId: 'questionsPrompt-2',
+    description:
+      'What isn’t clear about this proposal, decision, or idea? What needs further explanation?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Questions, Comments, Concerns❓', true),
+    question: 'Comments 💬',
+    description: 'What reactions do you have to this proposal, decision, or idea?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Questions, Comments, Concerns❓', true),
+    question: 'Concerns 🧐',
+    description: 'What worries do you have about this proposal, decision, or idea?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Team Retreat Planning 🌴', true),
+    question: 'Hopes 🙏',
+    description:
+      'What do you hope we will do together? What do you hope will come from our time together?',
+    sortOrder: 0
+  },
+  {
+    promptId: 'fearsPrompt-2',
```

Comment:
The prompt ids are created using the prompt question, but a few of the prompt questions are identical. I could have used a set and done this magically in the function, but manually adding it is faster for this migration.